client/components/count/index.jsx

```diff
@@ -2,22 +2,22 @@
  * External dependencies
  */
 import React from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
+import { localize } from 'i18n-calypso';
+import { identity, omit } from 'lodash';
 
-export default React.createClass( {
+const Count = ( { count, numberFormat, ...inheritProps } ) => (
+	<span className="count" { ...omit( inheritProps, [ 'translate', 'moment' ] ) }>
```

Comment:
Since it's not entirely obvious, I'd add a comment above this line explaining where the `translate` and `moment` prop are coming from. Something like

```js
// Omit props passed from the `localize` higher-order component that we don't need.
```